aspnetcore/toc.md

```diff
@@ -328,19 +333,20 @@
 ### [Enable QR code generation in Identity](xref:security/authentication/identity-enable-qrcodes)
 ### [Two-factor authentication with SMS](xref:security/authentication/2fa)
 ### [Use Cookie Authentication without Identity](xref:security/authentication/cookie)
-### [Azure Active Directory](xref:security/authentication/azure-active-directory/index)
-#### [Integrate Azure AD Into an ASP.NET Core web app](https://azure.microsoft.com/documentation/samples/active-directory-dotnet-webapp-openidconnect-aspnetcore/)
-#### [Integrate Azure AD B2C into a customer-facing ASP.NET Core web app](xref:security/authentication/azure-ad-b2c)
-#### [Integrate Azure AD B2C into an ASP.NET Core web API](xref:security/authentication/azure-ad-b2c-webapi)
-#### [Call a ASP.NET Core Web API from a WPF app using Azure AD](https://azure.microsoft.com/documentation/samples/active-directory-dotnet-native-aspnetcore/)
-#### [Call a Web API in an ASP.NET Core web app using Azure AD](https://azure.microsoft.com/documentation/samples/active-directory-dotnet-webapp-webapi-openidconnect-aspnetcore/)
+### Azure Active Directory
+#### [Overview](xref:security/authentication/azure-active-directory/index)
+#### [Integrate Azure AD Into a web app](https://azure.microsoft.com/documentation/samples/active-directory-dotnet-webapp-openidconnect-aspnetcore/)
```

Comment:
- Into --> into
- Can we shorten "Azure AD" to "AAD"?

Reply:
AAD is not allowed -- https://worldready.cloudapp.net/Styleguide/Read?id=2696&topicid=25369